### PR TITLE
Add primary key instead of unique on data_migrations table

### PR DESCRIPTION
**Pull request by aandis**
Closes https://github.com/ilyakatz/data-migrate/issues/151

### CODE REVIEW

**Comment by aandis** [comment]
cc: @ilyakatz

File: lib/data_migrate/data_schema_migration.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ def table_name
         ActiveRecord::Base.table_name_prefix + 'data_migrations' + ActiveRecord::Base.table_name_suffix
       end
 
-      def index_name
-        "#{table_name_prefix}unique_data_migrations#{table_name_suffix}"
+      def primary_key
```
Comment (ilyakatz) [inline lib/data_migrate/data_schema_migration.rb:9]: i've been out of Rails world for a while, what will this do?

Comment (ilyakatz) [reply lib/data_migrate/data_schema_migration.rb:9]: never mind, got it

**Comment by ilyakatz** [comment]
Thanks for this PR, @aandis. I'm trying to decide if this is a breaking change or not, so should it be a major or minor version change. What do you think?

**Comment by aandis** [comment]
@ilyakatz should be a minor version bump since it's not a breaking change. New installations of the gem will have the primary key via [`assure_data_schema_table`](https://github.com/ilyakatz/data-migrate/blob/7b36ac50dda0ae932ac54c7ba6f7b27eeb25fef8/lib/data_migrate/data_migrator.rb#L57) and existing installations would continue to work on the unique key. For existing installations, the [`assure_data_schema_table`](https://github.com/ilyakatz/data-migrate/blob/7b36ac50dda0ae932ac54c7ba6f7b27eeb25fef8/lib/data_migrate/data_migrator.rb#L57) [check](https://github.com/ilyakatz/data-migrate/blob/964d08a9ade9bab7269260dcb160f18cce034208/lib/data_migrate/data_migrator_five.rb#L12) would always pass so the table would not be recreated.

**Comment by ilyakatz** [comment]
ok, merged via #153

**Comment by ilyakatz** [comment]
new gem version released

**Comment by ilyakatz** [comment]
thanks for adding this @aandis

**Pull request closed without merging**